ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_order_statistics.py

```diff
@@ -10,6 +10,44 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# ptp
+@handle_frontend_test(
+    fn_tree="numpy.ptp",
+    dtype_values_axis=_statistical_dtype_values(function="ptp"),
+    where=np_frontend_helpers.where(),

```

Comment:
No need to generate where as there is no `where` argument in the function's signature

Reply:
Thanks for the comments. I'm a little confused on the `where` argument because i'm looking at the `nanpercentile` function in the same file and it uses the `where` command even though it's not in the function's signature. I guess i'm a little confused on what the argument is and how it's different in this situation.

Reply:
ok gotcha. since that parameter isn't in the numpy docs i took it out. anything else that needs fixing?